docs/standard/microservices-architecture/implement-resilient-applications/monitor-app-health.md

```diff
@@ -276,11 +276,11 @@ Finally, if you're storing all the event streams, you can use Microsoft Power BI
     [https://docs.microsoft.com/azure/service-fabric/service-fabric-health-introduction](/azure/service-fabric/service-fabric-health-introduction)
 
 -   **Azure Application Insights**
-    [https://azure.microsoft.com/services/application-insights/](https://azure.microsoft.com/services/application-insights/)
+    <https://azure.microsoft.com/services/application-insights/>
 
 -   **Microsoft Operations Management Suite**
-    [https://www.microsoft.com/en-us/cloud-platform/operations-management-suite](https://www.microsoft.com/en-us/cloud-platform/operations-management-suite)
+    <https://www.microsoft.com/en-us/cloud-platform/operations-management-suite>
```

Comment:
@mvelosop @CESARDELATORRE I think this also changed names. It redirects to a different site as well.